orangecontrib/timeseries/widgets/tests/test_owtabletotimeseries.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import numpy as np
+
+from Orange.data import Table
+from Orange.widgets.tests.base import WidgetTest
+
+from orangecontrib.timeseries import Timeseries
+from orangecontrib.timeseries.widgets.owtabletotimeseries import OWTableToTimeseries
+
+
+class TestAsTimeSeriesWidget(WidgetTest):
+    def setUp(self):
+        self.widget = self.create_widget(OWTableToTimeseries)  # type: OWTableToTimeseries
+
+    def test_nan_timeseries(self):
```

Comment:
Method name.

How do I know this test runs and passes?

Reply:
? Run the test?

Reply:
How do I see Travis hit it?